### PR TITLE
broadcast pie 🎄

### PR DESCRIPTION
**Pull request by JamesHageman**
@georgeutsin HELP! I can't css! It's all there, I just need you to fuck around with it till it works...

Happy Hannukah!

### CODE REVIEW

**Comment by JamesHageman** [comment]
Also the svg file isn't being loaded from netlify properly, so I need to figure that out

**Comment by JamesHageman** [comment]
FYI this PR assumes that #7 will go through :man_shrugging:

**Comment by JamesHageman** [comment]
K I fixed the svg thing, so please make sure you pull again @georgeutsin

File: src/Broadcasts/View.elm [hunk 1]
```diff
@@ -149,7 +149,7 @@ broadcastContent time broadcast =
                 0
 
         leftRotate =
-            if fractionTimeLeft < 5 then
+            if fractionTimeLeft < 0.5 then
```
Comment (georgeutsin) [inline src/Broadcasts/View.elm:130]: everything is terrible

File: src/Broadcasts/View.elm [hunk 2]
```diff
@@ -140,7 +140,7 @@ broadcastContent time broadcast =
     let
         fractionTimeLeft : Float
         fractionTimeLeft =
-            ((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour)
+            1 + (((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour))
```
Comment (georgeutsin) [inline src/Broadcasts/View.elm]: everybody hurts

Comment (JamesHageman) [reply src/Broadcasts/View.elm]: wait won't this always be greater than 1 then?

File: src/Broadcasts/View.elm (hunk 1 above)
Comment (JamesHageman) [reply src/Broadcasts/View.elm:130]: whoops

File: src/Broadcasts/View.elm (hunk 2 above)
Comment (georgeutsin) [reply src/Broadcasts/View.elm]: do some debugging/logging. `(Date.toTime broadcast.createdAt) - time` is actually the negative complement of the fraction of time left

Comment (JamesHageman) [reply src/Broadcasts/View.elm]: nvm I guess it works out

File: src/Broadcasts/View.elm [hunk 3]
```diff
@@ -140,7 +140,7 @@ broadcastContent time broadcast =
     let
         fractionTimeLeft : Float
         fractionTimeLeft =
-            1 + (((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour))
+            1 - ((time - (Date.toTime broadcast.createdAt)) / (24 * Time.hour))
```
Comment (JamesHageman) [inline src/Broadcasts/View.elm:121]: I think it makes sense doing `1 - stuff` to show that we're taking away from 1

Comment (georgeutsin) [reply src/Broadcasts/View.elm:121]: math is math

Comment (JamesHageman) [reply src/Broadcasts/View.elm:121]: also `time` is greater than `broadcast.createdAt` so I did mess up the order in the original equation

**Review by georgeutsin — APPROVED** [review]
lgtm 🐿

File: src/Broadcasts/View.elm [hunk 4]
```diff
@@ -113,28 +113,6 @@ broadcastRow f props b =
         ( key, html )
 
 
-
--- var rightRotate = props.fractionTimeLeft > 0.5 ? (props.fractionTimeLeft-0.5)*-360 : 0;
```
Comment (georgeutsin) [inline src/Broadcasts/View.elm]: 8be

**Comment by ethan-hardy** [comment]
🎅 🎅

**Pull request merged**